bitmark-references/bitmark-sdk/websocket.md

```diff
@@ -0,0 +1,85 @@
+# Web Socket
+Realtime event triggering is trending in modern application development nowadays. It helps the application is always up to date and so provides better user experience.
+Bitmark SDK offers Web Socket event triggering to help the application can be notified immediately when there are anything changes.
+
+## Event
+- New block: This event will be sent when bitmark blockchain has a new block.
+  + `blockNumber` : new block number
+- Bitmark changed: This event will be sent when users issue new bitmarks, received new bitmarks, or send them to new owner.
+  + `bitmarkId`: the id of bitmark is changed
+  + `txId`: the corresponding transaction id
+  + `presence`: indicate that the bitmark is owned or not by the corresponding `Account`
+- New transfer offer: This event will be sent when users receive new transfer offer (for 2 signatures transfer) from someone.
+  + `bitmarkId`: the id of bitmark is comming
+- New pending issuance: This event will be sent when users issue new bitmarks.
+  + `bitmarkId` : the id of bitmark is issuing
+- New pending transaction: This event will be sent to both sender and receiver.
+  + `txId`: the id of this pending transaction
+  + `owner`: the current owner of corresponding bitmark
+  + `prevTxId`: the previous id of this transaction
+  + `prevOwner`: the previous owner of corresponding bitmark
+
+Note: The attributes name is the sample from Bitmark Java SDK. There are something different in Swift SDK but the meaning is the same.
```

Comment:
I guess the only difference it could be either snakecase or camelcase. Any other possible difference?

Reply:
I haven't compared them in detail, but I guess it will be identical.

Reply:
Yes. The attributes meaning is the same in different implementation but i guess maybe the name convention or something is different. So the notice is just for helping people would not get confusing.

Reply:
> The actual query functions could be in snake case or camel case depends on which SDK is used. For simplicity, the camel case is used for explanation.

This is how I explain in the query page. FYI.